openslides/users/views.py

```diff
@@ -68,6 +71,7 @@ def update(self, request, *args, **kwargs):
             # Check permissions only to update yourself.
             if str(request.user.pk) != self.kwargs['pk']:
                 self.permission_denied(request)
+            # del request.data['groups_id']
```

Comment:
Here we need a solution that works together with the problem described here: https://github.com/OpenSlides/OpenSlides/issues/1871

I solved some of the problems but motion also also still open.